archunit/src/test/java/com/tngtech/archunit/library/dependencies/GraphTest.java

```diff
@@ -128,24 +132,37 @@ public void complete_graph() {
     @Test
     public void graph_which_causes_error_when_dependently_blocked_nodes_are_not_cleared_after_unblocking() {
         ImmutableSet<Integer> nodes = ImmutableSet.of(0, 1, 2, 3, 4, 5);
-        Graph<Integer, Integer> graph = new Graph<>();
+        Graph<Integer, Object> graph = new Graph<>();
         graph.addNodes(nodes);
 
         graph.addEdges(ImmutableSet.of(
-                new Edge<Integer, Integer>(0, 4),
-                new Edge<Integer, Integer>(1, 0),
-                new Edge<Integer, Integer>(1, 5),
-                new Edge<Integer, Integer>(2, 1),
-                new Edge<Integer, Integer>(3, 1),
-                new Edge<Integer, Integer>(3, 5),
-                new Edge<Integer, Integer>(4, 3),
-                new Edge<Integer, Integer>(5, 1),
-                new Edge<Integer, Integer>(5, 2)
+                newEdge(0, 4),
```

Comment:
Here, using `newEdge` is fine. Maybe it should be moved to `GraphTest`

Reply:
done